app/src/main/res/values/dimens.xml

```diff
@@ -26,4 +32,10 @@
     <dimen name="corner_rounding_5">18dp</dimen>
     <dimen name="corner_rounding_6">20dp</dimen>
 
+    <dimen name="editText_1">40dp</dimen>
```

Comment:
What are these for?

Reply:
If in the future we want to implement editText fields or larger editTextFields